packages/tizen_bundle/lib/tizen_bundle.dart

```diff
@@ -0,0 +1,332 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ffi';
+
+import 'package:ffi/ffi.dart';
+import 'package:flutter/services.dart';
+import 'package:tizen_interop/4.0/tizen.dart';
+import 'package:tizen_log/tizen_log.dart';
+
+/// The class for the property type of the bundle.
+class _BundleTypeProperty {
+  /// The array type.
+  static const int array = 0x0100;
+
+  /// The primitive type.
+  static const int primitive = 0x0200;
+
+  /// The measurable type.
+  static const int measurable = 0x0400;
+}
+
+/// The class for the type of the bundle.
+enum BundleType {
+  /// The none type.
+  none(0),
+
+  /// The string type.
+  string(1 | _BundleTypeProperty.measurable),
+
+  /// The string list Type.
+  strings(1 | _BundleTypeProperty.measurable | _BundleTypeProperty.array),
+
+  /// The bytes type.
+  bytes(2);
+
+  const BundleType(this.type);
+  final int type;
```

Comment:
How about `value` instead of `type` to avoid `type.type` in code.
```suggestion
  final int value;
```

Reply:
Done.